src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -727,6 +727,126 @@ class LedgerRPC_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testLedgerEntryCredentials()
+    {
+        testcase("ledger_entry Deposit Preauth with credentials");
+
+        using namespace test::jtx;
+
+        Env env(*this);
+        Account const issuer{"issuer"};
+        Account const alice{"alice"};
+        Account const bob{"bob"};
+        const char credType[] = "abcde";
```

Comment:
Why not store `strHex(std::string_view("abcde"))` instead of needing to recompute it every time?

Reply:
To not to duplicate the code